test/integration/aws/default/verify/controls/aws_flow_log.rb

```diff
@@ -0,0 +1,44 @@
+fixtures = {}
+%w[fl-vpc fl-subnet subnet_01_id ec2_security_group_default_vpc_id].each do |fixture_name|
+  fixtures[fixture_name] = attribute(
+    fixture_name,
+    default: "default.#{fixture_name}",
+    description: 'See ../build/flow_log.tf',
+  )
+end
+
+control 'aws_flow_log flow_log_id exists' do
```

Comment:
I know the naming stuff can seem pedantic, but without it, blocks like this can be really confusing.  Below, only the exists line is clear; everything else requires (for me, at least) a bit of mental gymnastics to understand what is being tested against what.  I'm dumb, don't make me think!